src/pages/public/login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment, useRef } from "react";
+import { useHistory } from "react-router-dom";
+import Card from "../../../components/Card/Card";
+import Navbar from "../../../components/Header/navbar";
+import styles from "./Login.module.css";
+const Login = (props) => {
+  let history = useHistory();
+
+  const receivedEmailID = useRef();
+  const receivedPassword = useRef();
+  const users = JSON.parse(localStorage.getItem("usersByEmail"));
+  // console.log(users);
+
+  const loginHandler = (e) => {
+    e.preventDefault();
+    const rxEmail = receivedEmailID.current.value;
+    const rxPassword = receivedPassword.current.value;
+
+    const user = users[rxEmail];
+    if (users[rxEmail]) {
+      if (user.password === rxPassword) {
+        console.log("Logged in");
+        // console.log(user);
+      }
+      history.push("/dashboard");
```

Comment:
this should be in the condition that you wrote in line 21

so if `entered password`is equal to the password stored in local storage then only you should move to dashboard